packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -452,6 +449,34 @@ const toggleOrientation = async ( driver ) => {
 	}
 };
 
+const isEditorVisible = async ( driver ) => {
+	const blockLocator = isAndroid()
+		? `//android.widget.EditText[contains(@content-desc, "Post title")]`
+		: `(//XCUIElementTypeScrollView/XCUIElementTypeOther/XCUIElementTypeOther[contains(@name, "Post title")])`;
+
+	await waitForVisible( driver, blockLocator );
+};
+
+const waitForVisible = async ( driver, blockLocator, iteration = 0 ) => {
```

Comment:
Since this function can wait for any element, we might consider renaming the ` blockLocator` argument to something more generic like `elementLocator`.